Mathlib/Data/Tree/Get.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2019 mathlib community. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro, Wojciech Nawrocki
+-/
+import Mathlib.Data.Num.Basic
+import Mathlib.Data.Tree.Basic
+
+#align_import data.tree from "leanprover-community/mathlib"@"ed989ff568099019c6533a4d94b27d852a5710d8"
+
+/-!
+# Binary tree get operation
+
+In this file we define `Tree.indexOf`, `Tree.get`, and `Tree.getOrElse`.
+These definitions were moved from the main file to avoid dependency on `Num`.
+
+## References
+
+<https://leanprover-community.github.io/archive/stream/113488-general/topic/tactic.20question.html>
```

Comment:
Let's link to the actual zulip discussion of this aspect, to save future readers some time searching:
```suggestion
<https://leanprover-community.github.io/archive/stream/113488-general/topic/tactic.20question.html#170999997>